roles/etcd/tasks/static.yml

```diff
@@ -54,11 +54,18 @@
   with_items:
   - etcd.yaml
 
+- name: create a directory for static pod
+  file:
+    state: directory
+    name: "{{ etcd_static_pod_location }}"
```

Comment:
This is misleading, this is where the disabled pods would be created.

Also, 755 is too broad, 0700 should be enough, same as in `Validate permissions on the static pods dir` task